greenplumpython/func.py

```diff
@@ -0,0 +1,36 @@
+from typing import Callable, Iterable, Optional
+from uuid import uuid4
+
+from .db import Database
+from .expr import Expr
+from .table import Table
+
+
+class FunctionCall(Expr):
+    def __init__(
+        self, func_name: str, db: Database, args: Iterable[Expr] = [], as_name: Optional[str] = None
+    ) -> None:
+        super().__init__(as_name)
+        self._func_name = func_name
+        self._args = args
+        self._db = db
+
+    def __str__(self) -> str:
+        args_string = ",".join([str(arg) for arg in self._args]) if any(self._args) else ""
+        return f"{self._func_name}({args_string})"
+
+    def to_table(self) -> Table:
+        as_string = f"AS {self._as_name}" if self._as_name is not None else ""
```

Comment:
![image](https://user-images.githubusercontent.com/449296/171137687-e79a40d0-1bf6-4e34-ad56-84a8cc474e38.png)

Out of the scope of this PR, the query string is hard to read when it gets more complex. We need to wrap lines & format it a bit.

Reply:
Agree. We can provide debuging functions for `Table` and `Expr`. 

pglast provides a function to prettify SQL: https://pglast.readthedocs.io/en/v3/api.html